stores/metadata.go

```diff
@@ -1180,9 +1211,18 @@ func (s *SQLStore) CopyObject(ctx context.Context, srcBucket, dstBucket, srcPath
 		if err != nil {
 			return fmt.Errorf("failed to fetch src slices: %w", err)
 		}
+		var slabIds []uint
 		for i := range srcSlices {
 			srcSlices[i].Model = Model{}  // clear model
 			srcSlices[i].DBObjectID = nil // clear object id
+			slabIds = append(slabIds, srcSlices[i].DBSlabID)
+		}
+
+		var srcSlabs []dbSlab
+		if err := tx.Where("id IN (?)", slabIds).
```

Comment:
This might result in a query too large to execute for certain objects. e.g. if it was uploaded with a part size of 5MiB and therefore references many slabs. It also fetches a lot of data from the db just for computing the health. 
Might be better to extend the object query with joins on slabs and slices and also select the health.

Reply:
It's a bit tricky to extend the object query, at least I don't know how to select an entity along with other fields, I tried but couldn't get it to work. I then tried using `s.object`, but that's gnarly, we would have to add several fields and make sure it gets hydrated properly. I ended up adding `objectHealth`, which I think is the simplest solution.